mypy/message_registry.py

```diff
@@ -80,6 +80,8 @@ def with_additional_msg(self, info: str) -> ErrorMessage:
 ASYNC_FOR_OUTSIDE_COROUTINE: Final = '"async for" outside async function'
 ASYNC_WITH_OUTSIDE_COROUTINE: Final = '"async with" outside async function'
 
+AWAIT_WITH_OUTSIDE_COROUTINE: Final = '"await" outside coroutine ("async def")'
```

Comment:
I'd prefer this message to be consistent with previous two: `"await" outside async function`, but probably that isn't worth changing an existing message.